app/dashboard/templates/ftux/job.html

```diff
@@ -90,7 +90,7 @@ <h2 class="text-center">{% trans "Job Opportunities" %}</h2>
   <div class="col-12 font-body mt-4">
     <label class="form__label label d-block" for="linkedinUrl">{% trans "Linkedin profile" %}</label>
     <div class="d-flex align-items-center">
-      <span>https://www.linkedin.com/in/</span><input name="linkedinUrl" id="linkedinUrl" value="{{profile.linkedin_url}}" class="form__input" type="text" placeholder="username" />
+      <span>https://www.linkedin.com/in/</span><input name="linkedinUrl" id="linkedinUrl" value="{% firstof profile.linkedin_url or '' %}" class="form__input" type="text" placeholder="username" />
```

Comment:
what does this fix?

Reply:
when I get into the job edition from was None as linkedin so If I didn't remove None before saving then my profile get with linkedin/None